Sources/Build/ToolProtocol.swift

```diff
@@ -12,6 +12,7 @@ import Basic
 import PackageModel
 import Utility
 import class Foundation.ProcessInfo
+import func SPMLibc.setenv
```

Comment:
This diff shouldn't be required anymore.

Reply:
done